plugins/emc/emc_vnx_block_lun_perfdata

```diff
@@ -0,0 +1,505 @@
+ longer than 15 characters.
+ Without these workarounds "Load" and "Queue Length" would not work.
+
+=head2 Installation
+
+ The plugin uses SSH to connect to Control Stations. It's possible to use 
+ 'nasadmin' user, but it would be better if you create read-only global user by
+ Unisphere Client. The user should have only Operator role. 
+ I created "operator" user but due to the fact that Control Stations already 
+ had one internal "operator" user, the new one was called "operator1". So be 
+ careful. After that, copy .bash_profile from /home/nasadmin to a newly created
+ /home/operator1.
+ 
+ On munin-node side choose a user which will be used to connect through SSH. 
+ Generally user "munin" is ok. Then, execute "sudo su munin -s /bin/bash", 
+ "ssh-keygen" and "ssh-copy-id" to both Control Stations with newly created 
+ user.
+ 
+ Make a link from /usr/share/munin/plugins/emc_vnx_dm_basic_stats to 
+ /etc/munin/plugins/emc_vnx_dm_basic_stats_<NAME>, where <NAME> is any 
+ arbitrary name of your storage system. The plugin will return <NAME> in its 
+ answer as "host_name" field.
+ 
+ For example, assume your storage system is called "VNX5300".
+ Make a configuration file at 
+ /etc/munin/plugin-conf.d/emc_vnx_block_lun_perfdata_VNX5300. For example:
+ 
+ [emc_vnx_block_lun_perfdata_VNX5300]
+ user munin					
+ env.username operator1
+ env.cs_addr 192.168.1.1 192.168.1.2	
+
+ or:
+
+ [emc_vnx_block_lun_perfdata_VNX5300]
+ user munin                                     
+ env.username operator1
+ env.localcli /opt/Navisphere/bin/naviseccli
+ env.sp_addr 192.168.0.3 192.168.0.4
+ env.blockpw foobar
+
+ Where: 
+ user - SSH Client local user
+ env.username - Remote user with Operator role for Block or File part
+ env.cs_addr - Control Stations addresses for remote (indirect) access.
+ env.localcli - Optional. Path of localhost 'Naviseccli' binary. If this 
+   variable is set, env.cs_addr is ignored, and local 'navicli' is used.
+   Requires env.blockpw variable.
+ env.sp_addr - Default is "SPA SPB". In case of "direct" connection to 
+   Storage Processors, their addresses/hostnames are written here.
+ env.blockpw - Password for connecting to Storage Processors
+
+=head1 ERRATA
+
+ It counts Queue Length in not fully correct way. We take parameters totally 
+ from both SP's, but after we divide them independently by load of SPA and SPB.
+ Anyway, in most AAA / ALUA cases the formula is correct.
+
+=head1 HISTORY
+
+ 09.11.2016 - First Release
+ 26.12.2016 - Compatibility with Munin coding style
+
+=cut
+
+export LANG=C
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+TARGET=$(echo "${0##*/}" | cut -d _ -f 6)
+
+# "All Storage Processors we have"
+if [[ -v "sp_addr" ]]; then
+	SPALL=$sp_addr
+else
+	SPALL="SPA SPB"
+fi
+# "navicli" command. Can be local or remote, through Control Stations
+if [[ -v "localcli" ]]; then
+	NAVICLI=$localcli
+else
+	NAVICLI="/nas/sbin/navicli"
+fi
+
+
+ssh_check_cmd() {
```

Comment:
Please add a comment regarding the (unusual) interface of this function.
(e.g. `prints "10" on stdout in case of success`)